docs/migration-guide-1.25-2.0.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Migration Guide 1.25-2.0
+---
+
+Once you upgrade to Kyma 2.0, perform the manual steps described in the Migration Guide.
+
+## Security
+
+### Native Kubernetes authentication in Kyma
+
+Kyma 2.0 simplifies things and switches to the native Kubernetes authentication. As a result, we no longer support the following authentication and authorization components:
+
+- API Server Proxy
+- Console Backend Service
+- Dex
+- IAM Kubeconfig Service
+- Permission Controller
+- UAA Activator
```

Comment:
this component is not installed on OS. do we have concept for migration guide for managed kyma? to not block this PR i would suggest to remove UAA Activator and think about migration guide for managed kyma.